CHANGELOG.md

```diff
@@ -0,0 +1,252 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
```suggestion
and since version 20.0.0 this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
and if we decide to do that, we should reflect it in the documentation, too. There is still a section that describes the old numbering scheme.